guides/doc-Managing_Hosts/master.adoc

```diff
@@ -68,5 +68,3 @@ include::common/assembly_job-template-examples-and-extensions.adoc[leveloffset=+
 
 [appendix]
 include::common/modules/ref_overview-of-the-host-columns.adoc[leveloffset=+1]
-
-endif::[]
```

Comment:
Is there a reason why this is removed?

Reply:
It came up as an error in my build.

Reply:
In my local build, your PR builds for all flavours without this change. Reading the `master.adoc`, it seems to be evenly split between `ifdef`+`ifndef` and `endif`.